qiskit/aqua/operators/gradients/natural_gradient.py

```diff
@@ -0,0 +1,426 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Natural Gradient. """
+
+import os
+from collections.abc import Iterable
+from typing import List, Tuple, Callable, Optional, Union
+
+import numpy as np
+from qiskit.aqua.operators import (OperatorBase, ListOp, ComposedOp, CircuitStateFn)
+from qiskit.aqua.operators.gradients.circuit_gradients import CircuitGradient
+from qiskit.aqua.operators.gradients.circuit_qfis import CircuitQFI
+from qiskit.aqua.operators.gradients.gradient import Gradient
+from qiskit.aqua.operators.gradients.gradient_base import GradientBase
+from qiskit.aqua.operators.gradients.qfi import QFI
+from qiskit.circuit import ParameterVector, ParameterExpression
+
+os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
```

Comment:
Could you add a comment on why this is required?

Reply:
It's redundant. I missed deleting it. Thanks for finding this.